map/map_tests/bookmarks_test.cpp

```diff
@@ -397,11 +397,12 @@ namespace
     search::AddressInfo info;
     fm.GetAddressInfoForGlobalPoint(MercatorBounds::FromLatLon(lat, lon), info);
 
-    TEST_EQUAL(info.m_name, poi.m_name, ());
     TEST_EQUAL(info.m_street, poi.m_street, ());
     TEST_EQUAL(info.m_house, poi.m_house, ());
-    TEST_EQUAL(info.m_types.size(), 1, ());
-    TEST_EQUAL(info.GetBestType(), poi.m_type, ());
+    // TODO(AlexZ): AddressInfo should contain addresses only. Refactor.
+    //TEST_EQUAL(info.m_name, poi.m_name, ());
```

Comment:
This struct is now used for addresses-only. It does not have information in name and types fields. It's planned refactoring.